crates/adapters/src/transport/kafka/ft/mod.rs

```diff
@@ -106,6 +106,12 @@ fn kafka_config(
     let default_redpanda_server = default_redpanda_server();
     set_option_if_missing(&mut settings, "bootstrap.servers", &default_redpanda_server);
 
+    // We link with openssl statically, which means that the default OPENSSLDIR location
+    // baked into openssl is not correct (see https://github.com/fede1024/rust-rdkafka/issues/594).
+    // We set the ssl.ca.location to "probe" so that librdkafka can find the CA certificates in a
+    // standard location (e.g., /etc/ssl/).
+    set_option_if_missing(&mut settings, "ssl.ca.location", "probe");
```

Comment:
https://docs.confluent.io/platform/current/clients/librdkafka/html/md_CONFIGURATION.html says:

> If OpenSSL is statically linked or ssl.ca.location is set to probe a list of standard paths will be probed and the first one found will be used as the default CA certificate location path.

so this should be the default anyway?

But the librdkafka documentation is often misleading or wrong, so if this change makes a difference then awesome, let's take it.

Reply:
Yeah, whatever rdkafka-rs does to configure openssl, this is not the default behavior it ends up with.